### ISSUE TITLE
Relations `narrower` and `instanceof`

### ISSUE THREAD

**Issue by Anaphory** [concepticon/concepticon-data#476]
(Followup from #469, #472, which were not well thought-out and showed my lack of understanding what the relations between concepts are for.)

It looks like you have the following four relationships:

1. Item from one word list where another word list felt the need to be semantically more precise, because two concepts co-lexified in the gloss language of the old list are not co-lexified in target languages of the new list, so it got noticed. (This is very apparent in most XXX OR YYY concepts and probably holds for most kinship terms, but also holds eg. for our FEMALE SARONG.) – *These items should be merged-and-marked when comparing concept lists, as narrower is now.*
2. Item from one word list does have a functional equivalent in a culture another word list is targeting, which is however different in significant features so that it's not just a narrowing of concept. I assume this holds for WOLF/DINGO/WILD DOG under WOLF-LIKE ANIMAL which I found above. From our word list, it's the case eg. for LOCUST/RICE EAR BUG (under *FIELD CROP PEST INSECT?) – *These items should be merged-and-marked to the overarching concept like narrower does now, but also if two lists contain cultural equivalent siblings, eg. merging DINGO and WILD DOG under WOLF-LIKE ANIMAL.* (Where the over-concept does not exist yet, or for speeding up computations, mark those pairs as cultural-replacement. I'm not sure what to do with eg. HEART,GUTS→LOCATION OF EMOTION here, merging HEART with GUTS just because they are both LOE in different cultures does not sound useful.)
3. Very specific narrowings of broad general concepts, which might all be met in the same culture. Like the ELF, FAIRY → LEGENDARY CREATURE, AUTUMN FESTIVAL, DRAGON BOAT FESTIVAL → FESTIVAL and SAGO PALM → PALM TREE mappings. – *These should be marked as connected (in particular given that they might not be in many English-based ontologies you might infer other relations from), but not merged when comparing word lists.*
4. Items that are ontologically somewhat narrower, but where there is no reason to assume confusion. You probably don't have too many of these in the database, because they are not really relevant for this application and better represented in external ontologies you link to. The examples I found could arguably be instances of cases 1 to 3 (eg. WALK → GO, HEADBAND → HEADGEAR, DEER → LARGE HERBIVORE). – *These may be marked as connected (although that can be offloaded to automatic inference from elsewhere)*

I would keep `narrower` for category **4** – and change it to something like `disambiguates` for category **1**.
I would fix all examples of category **3** to `instanceof` and make a note of what it means in a readme file somewhere, or think of a different gloss for that relation.
I would use something like `cultural-realization-of` (maybe something less clunky) for category **2** and make it behave like category 1 for now.

**Comment by LinguList** [comment]
ping @xrotwang, as we discussed this a lot in the past, could you maybe explain why we use narrower/broader (based on semantic web stuff, if I remember)? I don't have a particular opinion if this can be done quickly, as long as it doesn't break our tests and keeps our union and intersection commands intact.

**Comment by xrotwang** [comment]
The "narrower"/"broader" relations come from [SKOS](https://www.w3.org/TR/2009/NOTE-skos-primer-20090818/#sechierarchy) - and since we use SKOS concepts (and collections) to model the data in RDF (see e.g. http://concepticon.clld.org/parameters/2121.rdf), it seemed natural (in the sense of "most hope for being understood") to use these.

**Comment by Anaphory** [comment]
[From #472]
> I don't have the time nor the ability to go into all details about ontologies.

Formulated differently, my suggestion here is how to address the usefulness of the concept relations inside concepticon – namely, making concept lists comparable – without going full ontology. And therefore,
> Maybe deleting all my non-narrower/broader relations is even better (correcting for exemplar cases like "sago palm"), as it will make clear that we don't even try to go deeper (because we lack the capacities).

I agree with this in principle very much, with two notes given below.

> It was based on @xrotwang's suggestion, who had additional reasons for using the terms, as they are common in ontologies, etc.

I think that's a problem here, because it lead to at least my confusion that you want an ontology and not just disambiguation relations!

> In fact: we introduced the dingo-thing because they give "wolf/dingo" or things like that, so to keep their fuzzy over-arching concept, we introduced the other categories.

Yes, I noticed that. The fact that that broadness (‘wolf/dingo’) is obvious in their word list is a strong sign for me that this is an item of my category 2 above, not of category 1.

### Note:
The alternative to *deleting* the ontologically-narrower-but-not-in-the-word-list-sense concept relations would be to just *change their relationship type* – I'm always hesitant to throw away data. And for that case I thought `narrower` is actually more natural for that new relationship, wheras the remaining concepts feel more like `disambiguating` to me. But generally, as long as these things are handled differently it's fine, and that includes deleting all category 4 and keeping `narrower` for category 1.

My suggested `cultural-realization-of` relation could start as just an alias for `narrower`, and is admittedly not well thought-out yet. I just feel that there's a difference between these two categories, which might be useful down the line.

**Comment by LinguList** [comment]
> I think that's a problem here, because it lead to at least my confusion that you want an ontology and not just disambiguation relations!

We have this, however, already described in our paper in full. The paper dates from 2016, and we add this information there, so we can again put this on the website, make it clearer, but we'll always lead to some kind of confusion, and changing the relations in any kind of way will probably then confuse somebody else. Unless I wrongly understand, but if you suggest to rename broader-narrower since it is more technical, I am against it, as this was how we started in the past, and it would also require some change in code etc. Renaming other categories, or renaming broader-narrower exemplars which are wrong is fine with me. If we use, for example, your definitions, the name is not that important, and I would like to keep continuity in the names, also since we described it in papers and tutorials in different places.
